### ISSUE TITLE
CI should check the libraries as well as apps

### ISSUE THREAD

**Issue by bilalshaikh42** [reproducible-biomedical-modeling/CRBM-Viz#1572]
I believe the current setup for the ci does not run tests  or lint for the libraries, only for apps. This would cause issues in the libraries to be skipped.  Nx has a command to output the affected libraries as well that can be used.

**Comment by jonrkarr** [comment]
Done

**Comment by bilalshaikh42** [comment]
Seems to be some errors with libraries that do not have the test target.

**Comment by jonrkarr** [comment]
This can avoided either by adding the missing targets, even if there are no tests or running `nx affected:test` (but this would loose the parallelization).

**Comment by jonrkarr** [comment]
These missing targets should be caught now.

**Comment by bilalshaikh42** [comment]
> These missing targets should be caught now.

Is this in reference to https://github.com/biosimulations/Biosimulations/commit/30093e9ae5b1132c1bb88a937fd2b241dc65ef3f? Or was there some other change?

**Comment by jonrkarr** [comment]
I changed the workflow config `ci2.yml` so that it doesn't throw errors when it tries to lint or test targets that don't exist (i.e. libs that don't have a lint or test target defined).

```
{ ERROR_MSG=$(nx run ${{ matrix.libOrApp }}:test --code-coverage --codeCoverage 2>&1 1>&3-) ;} 3>&1
ERROR_CODE=$?
if [[ $ERROR_MSG =~ "An unhandled exception occurred: Project target does not exist." ]]; then
    echo "Target does not exist"
    exit 0
else
    echo "$ERROR_MSG"
    exit $ERROR_CODE
fi
```

**Comment by jonrkarr** [comment]
I forgot to push it. Its pushed now.
